app/controllers/callbacks_controller.rb

```diff
@@ -0,0 +1,11 @@
+class CallbacksController < Devise::OmniauthCallbacksController
+  def github
+    @user = User.from_omniauth(request.env["omniauth.auth"])
+    sign_in_and_redirect @user
+  end
+
+  def facebook
+    @user = User.from_omniauth(request.env["omniauth.auth"])
+    sign_in_and_redirect @user
+  end
+end
```

Comment:
Could make this more DRY by bringing the overlapping logic into a private method that both of these methods call